environments/custom/playbook-lvm.yml

```diff
@@ -1,5 +1,14 @@
+# https://docs.debops.org/en/stable-3.1/ansible/roles/lvm/defaults/main.html#volume-groups-and-logical-volumes
 - name: Manage LVM volumes
   hosts: generic
+  gather_facts: yes
   become: true
-  roles:
-    - role: debops.debops.lvm
+  tasks:
+    - name: Manage LVM Logical Volumes
+      community.general.lvol:
+        lv:         '{{ item.lv }}'
+        vg:         '{{ item.vg }}'
+        size:       '{{ item.size }}'
+        force:      '{{ item.force | d(omit) }}'
+        state:      'present'
```

Comment:
You should start to use yamllint. This will give you an "too many spaces after colon" error.

Reply:
Made some experiments today with yamllint and yamlfix.

Yaml checking throws currently a lot of errors and i haven't found a convenient way to fix them in a idempotent way. Therefore dropped my experiments form this branch and moved them to  https://github.com/SovereignCloudStack/hardware-landscape/pull/24.

Currently, I almost never have problems that are due to a defective formatting. Therefore, I will postpone this to a later date, as it is not that critical.